### DISCUSSION TITLE
Coordinates of the blocks found by ocr_predictor

### DISCUSSION THREAD

**Discussion by mat10599** [mindee/doctr#1329] [Q&A]
Hello, 

I would like to know if it's possible to get the coordinates of the results of the ocr_predictor.

```python

model = ocr_predictor('db_resnet50', 'crnn_vgg16_bn', pretrained=True)
doc = DocumentFile.from_pdf(doc.pdf")
result = model(doc)
```
The result I get is:

Block(
  (lines): [
    Line(
      (words): [Word(value='CREATIV'', confidence=0.99)]
    ),
    Line(
      (words): [
        Word(value='Agence', confidence=0.93),
        Word(value='d'ANNECY', confidence=0.99),
      ]
    ),
    Line(
      (words): [
        Word(value='109', confidence=0.92),
        Word(value='Avenue', confidence=1.0),
        Word(value='de', confidence=1.0),
        Word(value='Genève', confidence=0.98),
      ])

But I would also like to have the location of each word, line or block.
Is it possible to get this information ?

**Comment by felixdittrich92** [comment]
Hi @mat10599 👋 ,

See https://mindee.github.io/doctr/using_doctr/using_models.html#what-should-i-do-with-the-output

`result.export()`

Note: to restore the original coords you have to multiply the coresponding x and y values with the page dimension axis

```
    h, w = page["dimensions"]
    for block in page["blocks"]:
        for line in block["lines"]:
            for word in line["words"]:
                # Get aboslute word geometry
                (xmin, ymin), (xmax, ymax) = word["geometry"]
                xmin, xmax = int(round(w * xmin)), int(round(w * xmax))
                ymin, ymax = int(round(h * ymin)), int(round(h * ymax))
```
